mindsdb/integrations/handlers/elasticsearch_handler/elasticsearch_handler.py

```diff
@@ -263,26 +525,323 @@ def get_columns(self, table_name: Text) -> Response:
         Retrieves column (field) details for a specified table (index) in the Elasticsearch host.
 
         Args:
-            table_name (str): The name of the table for which to retrieve column information.
+            table_name (Text): The name of the table for which to retrieve column information.
+
+        Returns:
+            Response: A response object containing the column details.
 
         Raises:
             ValueError: If the 'table_name' is not a valid string.
+        """
+        if not table_name or not isinstance(table_name, str):
+            raise ValueError("Table name must be a non-empty string")
+
+        query = f"DESCRIBE {table_name}"
+        result = self.native_query(query)
+
+        if result.type == RESPONSE_TYPE.TABLE:
+            df = result.data_frame
+            df = df.drop("mapping", axis=1, errors="ignore")
+            result.data_frame = df.rename(columns={"column": "COLUMN_NAME", "type": "DATA_TYPE"})
+
+        return result
+
+    def get_column_statistics(self, table_name: str, column_name: Optional[str] = None) -> Response:
```

Comment:
This and the other data catalog related methods should start with `meta_`; `meta_get_column_statistics` and so forth. Once you inherit from one of `MetaDatabaseHandler` or `MetaAPIHandler`, the contract should be clear.